packit/config/common_package_config.py

```diff
@@ -182,6 +182,7 @@ def __init__(
         image_request: Optional[Dict] = None,
         image_customizations: Optional[Dict] = None,
         copr_chroot: Optional[str] = None,
+        follow_fedora_branching: bool = False,
```

Comment:
I am wondering if it makes sense to differentiate PR builds and branch/release ones?

Actually, we probably don't need to care since we will update the set of chroots if it does not fit, do we?